src/blurry_classifier_module.py

```diff
@@ -0,0 +1,210 @@
+        if "blurry_threshold" in config.attributes.fields:
+            self.blurry_threshold = config.attributes.fields["blurry_threshold"].number_value
+
+    # pylint: disable=too-many-arguments,too-many-positional-arguments
+    async def capture_all_from_camera(
+        self,
+        camera_name: str,
+        return_image: bool = False,
+        return_classifications: bool = False,
+        return_detections: bool = False,
+        return_object_point_clouds: bool = False,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> CaptureAllResult:
+        if camera_name not in (self.camera_name, ""):
+            raise ValueError(
+                "Camera name ",
+                {camera_name},
+                "does not match the camera name ",
+                {self.camera_name},
+                "in the config."
+            )
+        im = await self.camera.get_image(mime_type=CameraMimeType.JPEG)
+        classifications = None
+        if return_classifications:
+            classifications = await self.get_classifications(
+                im, 1, extra=extra, timeout=timeout
+            )
+
+        return CaptureAllResult(image=im, classifications=classifications)
+
+    async def get_detections_from_camera(
+        self,
+        camera_name: str,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> List[Detection]:
+        raise NotImplementedError()
+
+    async def get_detections(
+        self,
+        image: ViamImage,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> List[Detection]:
+        raise NotImplementedError()
+
+    async def get_classifications_from_camera(
+        self,
+        camera_name: str,
+        count: int,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> List[Classification]:
+        if camera_name not in (self.camera_name, ""):
+            raise ValueError(
+                "Camera name ",
+                {camera_name},
+                "does not match the camera name ",
+                {self.camera_name},
+                "in the config."
+            )
+        im = await self.camera.get_image(mime_type=CameraMimeType.JPEG)
+        return self.get_classifications(im, 1, extra=extra, timeout=timeout)
+
+    async def get_classifications(
+        self,
+        image: ViamImage,
+        count: int,
+        *,
+        extra: Optional[Mapping[str, ValueTypes]] = None,
+        timeout: Optional[float] = None,
+    ) -> List[Classification]:
+        img = decode_image(image)
+        blurriness = cv.Laplacian(img, cv.CV_64F).var()
+        if blurriness < self.blurry_threshold:
+            return [Classification(class_name="blurry", confidence=0.5)]
+
+        return []
+
+
```

Comment:
Sometimes you're separating functions with 1 blank line, sometimes 2. Be consistent. IIRC PEP8 says 1 blank line in between methods within a class, and 2 blank lines in between top-level things like class definitions.